app/src/main/res/values-sv/strings.xml

```diff
@@ -0,0 +1,200 @@
+
+Podcast search feature is planned for an update.
+
+PodListen source code is available at https://github.com/einmalfel/PodListen under GPL-3.0 license.
+If you encounter a bug or come up with a feature request, please open an issue on github.</string>
+
+    <string name="ok">OK</string>
+    <string name="cancel">Avbryt</string>
+
+    <!--only one-letter abbreviations will fit into layout. There is no room for longer ones-->
+    <string name="hour_abbreviation">h</string>
+    <string name="minute_abbreviation">m</string>
+
+    <string name="tab_playlist">Spellista</string>
+    <string name="tab_new_episodes">Nya episoder</string>
+    <string name="tab_subscriptions">Prenumerationer</string>
+
+    <string name="podcast_already_subscribed">Redan prenumererat på %s</string>
+    <string name="podcast_subscribe_failed">Felaktig prenumeration</string>
+    <string name="podcast_delete_question">Avbryta prenumeration av %s?</string>
+    <string name="podcast_delete_question_no_title">Avbryta perenumeration av denna podcast?</string>
+    <string name="podcast_no_title">Ingen titel</string>
+    <string name="podcast_refresh_failed">Uppdateringen falerade: %s</string>
+    <string name="podcast_not_loaded_yet">Strömmen är inte laddad än</string>
+    <string name="podcast_refresh_time">Uppdaterad %s</string>
+    <string name="podcast_image_description">Bild av denna podcast om det finns</string>
+    <string name="podcast_button_description">Visa episoder av denna ström</string>
+
+    <string name="episode_no_title">INGEN TITEL</string>
+    <string name="episode_downloading">Nedladding av episod %s</string>
+    <string name="episode_delete_question">Ta bort episod?</string>
+    <string name="episode_deleted">Episod borttagen: %s</string>
+    <string name="episode_deleted_undo">Ogör</string>
+    <string name="episode_image_description">Bild av denna episod om det finns</string>
+    <string name="episode_deleted_subscription">Borttagen prenumeration</string>
+    <string name="episode_published">Publicerad %s</string>
+
+    <string name="episode_action_download">Ladda ner episod</string>
+    <string name="episode_action_add">Addera denna episod till spellistan</string>
+    <string name="episode_action_play">Spela denna episod</string>
+    <string name="episode_action_pause">Pause</string>
+
+    <!--Ideally those strings should fit in single line in spinner widget (20-25 characters).
+    It's still ok if they don't fit, but it doesn't look neat. I'd recommend to exclude "add" word,
+    as it's usually longer in any language than in english. "Add" is already contained is spinner
+    title (see subscribe_dialog_on_first_load_text)-->
+    <string name="refresh_mode_week">Lägg till förra veckans episoder</string>
```

Comment:
These strings are a little bit too long. Maybe it's possible to remove "Lagg till" from them?